apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -43,15 +43,7 @@ export const TestWrapperDecoratorFullWidth: DecoratorFunction<ExtendedStoryFnRet
   </div>
 );
 
-const useNoAnimationStyles = makeStyles({
-  root: {
-    animationDuration: '0s !important',
-    transitionDuration: '0s !important',
-    '& *': {
-      animationDuration: '0s !important',
-      transitionDuration: '0s !important',
-    },
-  },
+const usePausedAnimationStyles = makeStyles({
   paused: {
     animationPlayState: 'paused !important',
     animationDelay: '-1s !important',
```

Comment:
Found the problem as to why it doesn't crop, you need to add flex to the styles so the div adapts to the content's height and width. 

```suggestion
const usePausedAnimationStyles = makeStyles({
  paused: {
    display: 'flex',
    animationPlayState: 'paused !important',
    animationDelay: '-1s !important',
```